float64 frictionSlide
float64 frictionDrag
float64 restitutionTable
---
bool success